src/main/java/pageObjects/Permissions.java

```diff
@@ -0,0 +1,52 @@
+package pageObjects;
+
+import driver.DriverFactory;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class Permissions extends DriverFactory {
+
+    public Permissions(WebDriver driver) {
+        super(driver);
+
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(id = "username")
+    private static WebElement usernameField;
+    @FindBy(id = "password")
+    private static WebElement passwordField;
+    @FindBy(id = "log-in-button")
+    private static WebElement signInButton;
+    @FindBy(id = "undefined-devices-mButton")
+    private static WebElement devicesButtonSidebarMenu;
+
+    public static void checkIfUserCanPerformActionOnDevicePage() throws InterruptedException {
+        driver.get("https://zenistock.zenitech.local/login");
+        usernameField.sendKeys("user");
+        passwordField.sendKeys("oiuhbvc5%r");
+        signInButton.click();
+        Thread.sleep(2000);
```

Comment:
remove thread.sleep and implement a smartwait method - https://www.browserstack.com/guide/wait-commands-in-selenium-webdriver

Reply:
also, could we use the credentials and links from another class? it would help with maintanance